src/mlpack/methods/bayesian_linear_regression/bayesian_linear_regression_main.cpp

```diff
@@ -0,0 +1,191 @@
+/**
+ * @file bayesian_linear_regression_main.cpp
+ * @author Clement Mercier
+ *
+ * Executable for BayesianLinearRegression.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <mlpack/prereqs.hpp>
+#include <mlpack/core/util/cli.hpp>
```

Comment:
```suggestion
#include <mlpack/core/util/io.hpp>
```
There was some change here recently in #2459; the `CLI` class changed to `IO`.  I'll provide suggestions below that should hopefully fix everything. :+1: